### ISSUE TITLE
ERROR - unable to resolve dependency tree

### ISSUE THREAD

**Issue by aabmets** [pixijs/pixi-react#354]
### Description
```
npm install @inlet/react-pixi
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: react-testing@0.1.0
npm ERR! Found: react@18.2.0
npm ERR! node_modules/react
npm ERR!   react@"^18.2.0" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^17.0.1" from @inlet/react-pixi@6.8.0
npm ERR! node_modules/@inlet/react-pixi
npm ERR!   @inlet/react-pixi@"*" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

### Steps to reproduce

1. `npm install @inlet/react-pixi`

### Additional info

- `@inlet/react-pixi` version: 6.8.0
- `React` version: 18.2.0
- `ReactDOM` version: 18.2.0
- `PIXI` version: 6.5.6

**Comment by designspin** [comment]
Looks like the issue is in hand though: https://github.com/inlet/react-pixi/pull/338 , great stuff!

**Comment by baseten** [comment]
This is resolved in the latest version `v7.0.0-alpha.0`, this now lives under `@pixi/react-pixi` on npm
